scala/lambdas/preingest-tdr-aggregator/src/main/scala/uk/gov/nationalarchives/preingesttdraggregator/Duration.scala

```diff
@@ -17,10 +17,12 @@ object Duration:
     def *(other: Int): Int = s * other
     def length: Int = s
     def toJson: Json = Json.fromInt(s)
+    def toMilliSeconds: MilliSeconds = s * 1000
```

Comment:
I just realised that the S is capitalised; could you lowercase it?

Reply:
Ah yeah. I was thinking milli seconds as two words but it's not :facepalm: I've changed it.